pkg/egg/parse_testcase.go

```diff
@@ -0,0 +1,70 @@
+package egg
+
+import "github.com/aquasecurity/go-dep-parser/pkg/types"
+
+var (
```

Comment:
since `egg-info` is old format, I haven't been able to find a way to diversify the test cases, so most of these are actually redundant.
it also seems that `trivy` picks up library based on the current language version? I ran a scan and `trivy` only checks for `python3` packages